pwaCustomButton.js

```diff
@@ -0,0 +1,19 @@
+if (typeof window !== 'undefined') {
+  const pwaBtn = document.querySelector('#pwa-button');
```

Comment:
Move this code into the listener below.

Reply:
This need to be before `beforeinstallprompt` listener since if the app is installed, I do not want `#pwa-button` to be displayed

Reply:
Set a `hidden` attribute on it, and only show it if you need to.

Reply:
Fixed